src/components/RadioTile/RadioTile.js

```diff
@@ -29,9 +31,14 @@ export default class RadioTile extends React.Component {
   render() {
     const { children, ...other } = this.props;
 
-    const classes = classNames('bx--tile', 'bx--tile--selectable', {
-      'bx--tile--is-selected': this.props.checked,
-    });
+    const classes = classNames(
+      this.props.className,
```

Comment:
Could add `className` to the `const { children, ...other } = this.props` de-structure statement above 🙂

Reply:
will do 👍